Add command to toggle RTS

planteen opened the pull request:
Toggle RTS: this is a new command that parallels the command for toggling DTR. Toggling RTS is useful for troubleshooting flow control signals. It is also useful for embedded hardware which is configured so RTS controls an external component (such as a relay, FET, or enable line). Toggling RTS is supported if the flow control mode is none or xon/xoff (not supported under RTS/CTS flow control for obvious reasons). The TIOCMBIC and TIOCMBIS ioctls() on TIOCM_RTS under Linux still succeed if the flow control is RTS/CTS, but the actual state on the RTS line does not change.

npat-efault commented on term.c:
```diff
@@ -1440,6 +1444,105 @@ term_lower_dtr(int fd)
+				break;
+			}
+		}
+#else
```
Why this else block for non-linux systems?

npat-efault commented on term.c:
```diff
@@ -1440,6 +1444,105 @@ term_lower_dtr(int fd)
+				break;
+			}
+		}
+#else
```
Again, why this else-block for non-Linux?
In the raise/lower DTR case, the portable way to do it is to set the baudrate to 0 (B0). 
I don't think this is true for RTS

planteen commented:
I looked through BSD documentation about termios, but didn't find anything about how to control RTS. So I modified the code so this command is only supported in Linux. I also updated the man page and fixed a few typos and spelling errors in it. Let me know if you had something else in mind.

npat-efault commented:
Merged, thanks!
